config.json

```diff
@@ -128,6 +134,16 @@
             "MsBuildLogging":"/flp:v=normal;LogFile=build-packages.log"
           }
         },
+        "sign": {
```

Comment:
could you please call this signpackages so it is clearer.

Reply:
Sure, done